spec/fixtures/example_config/invalid_config.yml

```diff
@@ -0,0 +1 @@
+this is not a yaml file!!
```

Comment:
It doesn't look like this file is getting used anywhere. Do we need it?

Reply:
correct! fixing